src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -3967,6 +3966,8 @@ namespace EnergyPlus::ZoneEquipmentManager {
                                     state.dataZoneEquip->ZoneEquipConfig(ZoneNum).TotInletAirMassFlowRate + MassConservation(ZoneNum).MixingSourceMassFlowRate);
                     }
                     CalcZoneMixingFlowRateOfReceivingZone(ZoneNum, ZoneMixingAirMassFlowRate);
+                    ZoneMixingNetAirMassFlowRate = MassConservation(ZoneNum).MixingMassFlowRate - MassConservation(ZoneNum).MixingSourceMassFlowRate;
+
```

Comment:
Calculates update the mixing objects net zone mass flow rate for immediate use, instead of waiting until the next iteration.